tool/tctl/common/resource_command.go

```diff
@@ -817,7 +893,13 @@ func (rc *ResourceCommand) createLoginRule(ctx context.Context, client auth.Clie
 	_, err = loginRuleClient.CreateLoginRule(ctx, &loginrulepb.CreateLoginRuleRequest{
 		LoginRule: rule,
 	})
-	return trail.FromGRPC(err)
+
+	if err != nil {
+		return trace.AlreadyExists("login rule '%s' already exists, use -f flag to override", rule.Metadata.GetName())
+	}
```

Comment:
I think `createLoginRule` was already close to fine [as-is](https://github.com/gravitational/teleport/blob/6fdc2fd1df795088466bbe75e48d1fefa1514d62/tool/tctl/common/resource_command.go#L804-L821), presuming that `CreateLoginRule` returns `trace.AlreadyExists` if the login rule already exists. It could just use a `fmt.Printf` at the bottom. (Please manually double check me on that, though).

Regardless, this line doesn't make much sense. That `err` could be anything, not just a `trace.AlreadyExists`.